ts/api/pagopa.ts

```diff
@@ -30,6 +34,42 @@ import {
   BasicResponseTypeWith401
 } from "./backend";
 
+export function emptyResponseDecoder<
+  S extends number,
+  H extends string = never
+>(status: S): ResponseDecoder<IResponseType<S, undefined, H>> {
+  return async (response: Response) => {
+    if (response.status !== status) {
+      return undefined;
+    }
+    const text = await response.text();
+    if (text !== "") {
+      return undefined;
```

Comment:
perhaps we should be less strict here? this means that if the response has some content in the body, the decoding will fail altogether.

Reply:
This is an **empty**ResponseDecoder, so I would expect the response to be empty. As I mentioned [yesterday](https://developersitalia.slack.com/archives/CA70BM37X/p1537449393000100), we could have the `basicResponseDecoder` handle cases where the `Content-Type` is not json, and in this specific case here we could use `basicResponseDecoder<EmptyString>`